tests/unit/modules/test_disk.py

```diff
@@ -99,6 +99,13 @@ def test_blkid(self):
                 patch('salt.modules.disk.blkid', MagicMock(return_value=STUB_DISK_BLKID)):
             self.assertDictEqual(STUB_DISK_BLKID, disk.blkid())
 
+    def test_blkid_token(self):
```

Comment:
@aplanas Looks like these tests fail on windows, skip them?

Reply:
Ouch. Yes!

The real problem is the interaction with the decorator in the disk module, that search for the blkid binary. I think that is better to skip than to add a new mock for this decorator.

Thanks!